src/conn.c

```diff
@@ -72,9 +72,9 @@ static void gateway_handle_cb(struct handle *req, int status, int type)
 	n = buffer__offset(&c->write) - message__sizeof(&c->response);
 	assert(n % 8 == 0);
 
-	c->response.type = (uint8_t)type;
+	c->response.type = type;
 	c->response.words = (uint32_t)(n / 8);
-	c->response.schema = 0;
+	c->response.schema = schema;
```

Comment:
Useful to assert that the response schema is equal or less than the `req->schema`?